src/logic/operate.js

```diff
@@ -0,0 +1,33 @@
+import Big from 'big-js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const operations = {
+    sum: '+', subtract: '-', multiplication: 'x', division: '÷', percentage: '%',
+  };
+  const x = Big(numberOne);
+  const y = Big(numberTwo);
+  let result = null;
+  switch (operation) {
+    case operations.sum:
+      result = x.plus(y);
+      break;
+    case operations.subtract:
+      result = x.minus(y);
+      break;
+    case operations.multiplication:
+      result = x.times(y);
+      break;
+    case operations.division:
+      result = x.div(y);
+      break;
```

Comment:
dont forget to manage DIVION BY ZERO on next milestone, else it will generate an error